src/test/java/uk/gov/ch/oauth/Oauth2Test.java

```diff
@@ -42,6 +23,15 @@
 import uk.gov.companieshouse.session.model.SignInInfo;
 import uk.gov.companieshouse.session.store.Store;
 
+import javax.servlet.http.HttpServletRequest;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+
+import static org.junit.Assert.*;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.mockito.Mockito.*;
+
```

Comment:
These changes seem to be causing merge conflicts. I'm not quite why there have been import changes on this file, maybe a merge went wrong somewhere?

Reply:
Im not sure, I tried changing and committing but it wouldn't let me

Reply:
Should I try and resolve it on the web-editor below?

Reply:
Yeah, could be the way to go. What's happening when you try to make the changes on your local editor?

Reply:
I made the changes and added a space somewhere and it let me push but for some reason it did not change the imports. Mabye Intellij "cleans" them if it notices multiple imports from the same package to a *

Reply:
There might be something with you commit settings. Mine is set to optimise imports and cleanup the code. Are you able to see what your settings are?

Reply:
Ive unchecked optimise imports and its cleared some of them

Reply:
I you manually revert the imports to how they supposedly were before all your changes and then commit and push again without optimise imports.....or is that what you've just done?

Reply:
I then committed with each one imported, I think the code conventions said don't use .* in imports and then committed and it cleared it. There is another part that is conflicting which is not on my local version. I think that may be the new stuff Zain merged in to /ci

Reply:
Have you merged Zain's most recent changes into your current branch?

Reply:
That worked :)